### PR TITLE
VB-245: Feature/create product routes

### PR DESCRIPTION
**Pull request by mishabenq**

### CODE REVIEW

**Review by Nightlngale — APPROVED** [review]

File: packages/backend/src/api/routes/product.ts [hunk 1]
```diff
@@ -247,13 +251,81 @@ export const initProductRoutes = (
    */
 
   router.get(
-    apiPath(path, ProductApiRoutes.ID),
-    wrap((req: Request) => productService.getById(req.params.id)),
+    apiPath(path, ProductApiRoutes.SIMILAR),
```
Comment (Nightlngale) [inline packages/backend/src/api/routes/product.ts:254]: FYI: in rest it's better to have products/id/similar

File: packages/backend/src/repositories/product.ts [hunk 2]
```diff
@@ -224,4 +224,45 @@ export class ProductRepository {
       },
     });
   }
+
+  public async findSimilar(
+    city: string,
```
Comment (voonya) [inline packages/backend/src/repositories/product.ts:229]: Maybe remove city like option of similarity?

Comment (mishabenq) [reply packages/backend/src/repositories/product.ts:229]: it was in ticket requirements, but i can remove it if its necessary

Comment (voonya) [reply packages/backend/src/repositories/product.ts:229]: Ok, just interested

**Review by voonya — APPROVED** [review]

File: packages/backend/src/repositories/product.ts (hunk 2 above)
Comment (Nightlngale) [reply packages/backend/src/repositories/product.ts:229]: @voonya, are you speaking about not sending "city" from FE and take it by "productId"?

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**